modules/oe_webtools_media/tests/src/FunctionalJavascript/MediaSourceWebtoolsTest.php

```diff
@@ -32,52 +32,56 @@ class MediaSourceWebtoolsTest extends MediaSourceTestBase {
   ];
 
   /**
-   * Tests the webtools media source.
-   *
-   * @param string $widget_type
-   *   The widget type.
-   * @param string $widget_name
-   *   The widget human name.
-   * @param string $service
-   *   The webtools service associated with the widget.
-   * @param string $thumbnail_filename
-   *   The filename of thumbnail.
-   *
-   * @dataProvider providerTestMediaWebtoolsSource
+   * {@inheritdoc}
    */
-  public function testMediaWebtoolsSource(string $widget_type, string $widget_name, string $service, string $thumbnail_filename): void {
-    $media_type_id = 'test_media_webtools_type';
+  protected $defaultTheme = 'classy';
 
+  /**
+   * Tests the webtools media source.
+   */
+  public function testMediaWebtoolsSource(): void {
```

Comment:
This is a test refactor as we were using a data provider in a JS functional test which would take forever. So I looped over the data set.